features/salt_formulas.feature

```diff
@@ -57,6 +69,14 @@ Feature: Use salt formulas
      And I click on "Clear values" and confirm
      And I click on "Save Formula"
      Then I should see a "Formula saved!" text
+     And I refresh the pillar data
+     And the pillar data for "formulas" should be "- locale" on "sle-minion"
+     And the pillar data for "timezone:name" should be "CET" on "sle-minion"
+     And the pillar data for "keyboard_and_language:keyboard_layout" should be "English (US)" on "sle-minion"
+     And the pillar data for "keyboard_and_language:language" should be "English (US)" on "sle-minion"
+     And the pillar data for "formulas" should be empty on "ceos-minion"
+     And the pillar data for "timezone" should be empty on "ceos-minion"
```

Comment:
Thanks for your feedback @MalloZup ! 👍 

Would you be ok if we move the `salt_formula.feature` to be executed after `mincentos7` is bootstrapped? That way we would prevent from repeating all the formula stuff to test leaking on that new `salt_formula_leaks.feature`.

I would move `salt_formula.feature` after bootstraping centos7, as we can consider it now as dependent of both minions (sle-minion and ceos-minion).

What do you think?